cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -3,12 +3,51 @@ package infra
 type AzureResourceType string
 
 const (
```

Comment:
Can we make this dynamic so we don't have to update CLI to get this map

Reply:
Unfortunately, not right now -- this is noted in the comments for the code:

// Azure Resource Manager does not offer an API for obtaining display name for resource types.
// Display names for Azure resource types in Azure Portal are encoded in UX definition files instead.
// As a result, we provide static translations for known resources below. These are obtained from the Azure Portal.

Reply:
Can you please include a mapping file that has all resources and their friendly names (or the popular ones)?  The reasoning is that 

1. devs will build templates with any resources and I'd like to have as much coverage as possible.
2. provisoining one resource will sometimes provision other resources that we haven't accounted for, could another type that isn't explicitly in the template

Reply:
Created #180